src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -473,6 +468,11 @@ internal static ICapabilities SetPlatformToCapabilities(DriverOptions dc, string
             return result.AsReadOnly();
         }
 
+        protected static Dictionary<string, object> PrepareArgument(string param, object value)
```

Comment:
is it possible to have a method with variable number of params and values?

Reply:
Do you mean sth like:
`PrepareArguments(key1, value1, key2, value2, ...)`?

Reply:
yes.  An alternative to `ImmutableMap.of(....)` in Google Guava's implementation

Reply:
Then yes, I think it's possible. I'll try to do it

Reply:
@akinsolb You're right! Awesome! Then I can use it and remove my method. Nice. On it.

Reply:
Done